pkg/env/env.go

```diff
@@ -17,8 +17,9 @@ const whiteSpaces = " \t"
 // DefaultEnvVariables returns a default environment, with $PATH and $TERM set.
 func DefaultEnvVariables() map[string]string {
 	return map[string]string{
-		"PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
-		"TERM": "xterm",
+		"PATH":      "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+		"TERM":      "xterm",
+		"container": "podman",
```

Comment:
Will tvis work,if the OCI image has explicit PATH env variable with exact this value?

Reply:
These environment variables will not show up in the kube.yaml file.  If the image has these fields, then the container should still get them when kubernetes runs the image.